src/effects/VST/VSTEffect.cpp

```diff
@@ -1005,17 +1005,32 @@ namespace
 {
    struct VSTEffectMessage : EffectInstance::Message
    {
-      explicit VSTEffectMessage(VSTEffectSettings settings)
-         : settings{ std::move(settings) }
-      {}
+      typedef std::vector<std::optional<double> > ParamVector;
+
+      // Make a message from a chunk and ID-value pairs
+      explicit VSTEffectMessage(const std::vector<char>& chunk, const ParamVector& params)
```

Comment:
Change `&` to `&&` twice

Reply:
OK, and I left the const modifier, not sure if it makes a difference but it does not hurt

Reply:
Remove the const modifiers